feature-libs/product-configurator/rulebased/components/overview-bundle-attribute/configurator-overview-bundle-attribute.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, Input, OnInit } from '@angular/core';
+import {
+  ImageGroup,
+  Product,
+  ProductScope,
+  ProductService,
+} from '@spartacus/core';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Configurator } from '../../core/model/configurator.model';
+import { ConfiguratorPriceComponentOptions } from '../price/configurator-price.component';
+
+@Component({
+  selector: 'cx-configurator-cpq-overview-attribute',
+  templateUrl: './configurator-overview-bundle-attribute.component.html',
+})
+export class ConfiguratorOverviewBundleAttributeComponent implements OnInit {
+  product$: Observable<Product>;
+
+  @Input() attributeOverview: Configurator.AttributeOverview;
+
+  constructor(protected productService: ProductService) {}
+
+  ngOnInit() {
+    const noCommerceProduct: Product = { images: {} };
+    if (this.attributeOverview?.productCode) {
+      this.product$ = this.productService
+        .get(this.attributeOverview?.productCode, ProductScope.LIST)
+        .pipe(
+          map((respProduct) => {
+            return respProduct ? respProduct : noCommerceProduct;
+          })
+        );
+    } else {
+      this.product$ = of(noCommerceProduct);
```

Comment:
this `noCommerceProduct` make `*ngIf="product$ | async as product` always true. Do we want to display product info if product not exist?

Reply:
Yes. In CPQ we can have products that don't exist in commerce but still are valid bundle elements (and sent to S/4 e.g.). Not the best modeling, as we don't have pictures then, but allowed